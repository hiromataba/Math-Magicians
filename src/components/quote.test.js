import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './quote';

it('expect quote to match snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
