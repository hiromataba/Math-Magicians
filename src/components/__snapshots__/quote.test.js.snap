// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match snapshot 1`] = `
<div>
  <p
    className="quote"
    style={
      Object {
        "color": "#363c48",
        "fontSize": "34px",
        "left": "50%",
        "paddingBottom": "150px",
        "position": "absolute",
        "top": "50%",
        "transform": "translate(-50%,-50%)",
      }
    }
  >
    Mathmatics is not about numbers, equations, compulations, or algorithms: It is about understanding. -William Paul Thurstan
  </p>
</div>
`;
